src/sort_bed.py

```diff
@@ -1,11 +1,12 @@
 """Tool for cleaning up a BED file."""
 
-import argparse  # we use this module for option parsing. See main for details.
+import argparse
+from operator import attrgetter  # we use this module for option parsing. See main for details.
 
 import sys
-from typing import TextIO
+from typing import List, TextIO
```

Comment:
You should use `list` instead of `List`. The former works without importing from `typing` (in a modern Python) and the second is deprecated and will be removed in the next major release.